tests/ssl-opt.sh

```diff
@@ -11473,6 +11557,7 @@ requires_config_enabled MBEDTLS_DEBUG_C
 requires_config_enabled MBEDTLS_SSL_CLI_C
 requires_all_configs_enabled MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE \
                              MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_EPHEMERAL_ENABLED
+requires_pk_alg "ECDSA"
```

Comment:
I think for this case, and probably all the other TLS 1.3 cases until the end of the files, `detect_require_features` already does the job so don't need this explicit declaration.